thumbmark-android/src/main/java/com/thumbmarkjs/thumbmark_android/models/Fingerprint.kt

```diff
@@ -5,12 +5,12 @@ import kotlinx.serialization.Serializable
 
 @Serializable
 data class Fingerprint(
-    var captureDevices: List<CaptureDevice>,
-    var processor: Processor,
-    var device: Device,
-    var memory: Memory,
-    var accessibility: Accessibility,
-    var locality: Locality,
-    var communication: Communication,
+    var captureDevices: List<CaptureDevice>?,
```

Comment:
noticed this on the swift one as well, this isn't really composable if you need to edit this file to add new ones, something to consider for future iterations.